projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -977,6 +987,21 @@ function wpcom_launchpad_is_woocommerce_setup_visible() {
 	return in_array( 'woocommerce/woocommerce.php', $active_plugins, true );
 }
 
+/**
+ * Determines wheter or not the sensei setup task should be visible.
+ *
+ * @return bool True if the sensei setup task should be visible.
+ */
+function wpcom_launchpad_is_sensei_setup_visible() {
+	$is_atomic_site = ( new Automattic\Jetpack\Status\Host() )->is_woa_site();
+	if ( ! $is_atomic_site ) {
+		return false;
+	}
+
+	$active_plugins = get_option( 'active_plugins' );
```

Comment:
Would it be a little cleaner to use `is_plugin_active` instead? Not sure if that will work in this context.

Reply:
In this case, the option actually returns all the active plugins. Perhaps it would be clearer if we did something like:
```suggestion
	$active_plugins = get_option( 'active_plugins' );
	$is_sensei_active = in_array( 'sensei-lms/sensei-lms.php', $active_plugins, true );
	return $is_sensei_active;
```
WDYT?

Reply:
What I'm wondering is, can we do this?

```
return is_plugin_active( 'sensei-lms/sensei-lms.php' ); 
```

Then we can avoid calling `get_option` entirely. Again, I'm not sure if this is possible but i was curious because it would

Reply:
Oh, great! I didn't know about this function, but I will try to use it.